packages/components/src/navigable-container/container.js

```diff
@@ -99,7 +99,14 @@ class NavigableContainer extends Component {
 			// 'handling' the event, as voiceover will try to use arrow keys
 			// for highlighting text.
 			const targetRole = event.target.getAttribute( 'role' );
-			if ( MENU_ITEM_ROLES.includes( targetRole ) ) {
+			const targetHasMenuItemRole =
+				MENU_ITEM_ROLES.includes( targetRole );
+
+			// `preventDefault()` on tab to avoid having the browser move the focus
+			// after this component has already moved it.
+			const isTab = event.code === 'Tab';
+
+			if ( targetHasMenuItemRole || isTab ) {
```

Comment:
Hmm, so if I understand correctly, this component kind of assumes that all navigable children will have a menuitem* role? 🤔 So in a NavigableMenu, the "prevent scroll containers from scrolling" part will not work if children don't have a menuitem* role. This seems like an important detail that is yet undocumented.

Reply:
It looks like it — it was a behaviour that existed before this PR, and it was not documented (it looks like `NavigableMenu` is not used frequently in the codebase, and `TabbableContainer` is not used at all).

I will add some documentation about it

Reply:
Done in https://github.com/WordPress/gutenberg/pull/43606/commits/9c38277ac2bf2c695e6003abac4d2860327364ea

I explicitly didn't mention the `preventDefault` behaviour explicitly because it felt like a bit of an implementation detail, which I didn't want to expose in official docs.